### ISSUE TITLE
disable messages on vote

### ISSUE THREAD

**Issue by OlafHaag** [kolar/telegram-poll-bot#5]
This is not a bug, but for convenience. I tried the simplepollbot and everyone in our group was annoyed by the messages the bot sends. So I'm asking, if there could be an option, where the messages from the bot on new votes or vote changes are disabled? Unfortunately I lack the php skils to fully understand the code. And I'm not sure which parts I would have to rip out and not cause any errors. Lines 343-357 in Pollbot.php look like good candidates.

**Comment by Kimischa** [comment]
Agree, the group get's quiet messy with all the messages from the bot!

**Comment by ghost** [comment]
Another +1, this bot could be a little less noisy

**Comment by siiiia** [comment]
Me too.

**Comment by aWeinzierl** [comment]
I agree, exactly this prevents me from using that bot.
